startleft/api/controllers/iac/iac_create_otm_controller.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from http import HTTPStatus
+
+from fastapi import APIRouter, File, UploadFile, Form, Response
+
+from startleft.api.error_response import ErrorResponse
+from startleft.messages import messages
+from startleft.project.otm_project import OtmProject
+
+PREFIX = '/api/v1/startleft/iac'
+URL = ''
+RESPONSE_STATUS_CODE = HTTPStatus.CREATED
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix=PREFIX,
+    tags=["IaC"],
+    responses={
+        201: {"description": messages.OTM_SUCCESSFULLY_CREATED},
+        400: {"description": messages.BAD_REQUEST,
+              "model": ErrorResponse},
+        401: {"description": messages.UNAUTHORIZED_EXCEPTION,
+              "model": ErrorResponse},
+        403: {"description": messages.FORBIDDEN_OPERATION,
+              "model": ErrorResponse},
+        'default': {"description": messages.UNEXPECTED_API_ERROR,
+                    "model": ErrorResponse}
+    }
+)
+
+
+@router.post(URL, status_code=RESPONSE_STATUS_CODE)
```

Comment:
Add a little description to be shown in the Swagger.

`@router.post(URL, status_code=RESPONSE_STATUS_CODE, description="Generates an OTM threat model from an IaC file")`

It will be displayed here:
![image](https://user-images.githubusercontent.com/99404665/159273392-a43564b1-92db-4186-aad4-ffd4024132ed.png)